src/cli.rs

```diff
@@ -158,6 +159,5 @@ mod tests {
             ..Args::default()
         };
         assert_eq!(args, parse("cargo sweep --toolchains 1,2,3").unwrap());
-        assert!(parse("cargo sweep --toolchains 1 2 3").is_err());
```

Comment:
Can you add a test that the equivalent with "1 2 3" as a single argument (instead of using split_whitespace) is still an error?

Reply:
Oh right, this is https://github.com/rust-lang/rustup/issues/3168 again. Ok, no need for the new test.